apps/src/templates/studioHomepages/teacherHomepageV2/SectionCardBody.tsx

```diff
@@ -20,18 +20,16 @@ interface SectionCardBodyProps {
 const SectionCardBody: React.FC<SectionCardBodyProps> = ({section}) => {
   return (
     <div className={styles.sectionCardBody}>
-      <div className={styles.sectionCardBodyLeft}>
```

Comment:
if you are getting rid of this `div` can you get rid of the associated `sectionCardBodyLeft` styling??